### PR TITLE
Add PowerShell module [WIP]

### PR DESCRIPTION
**Pull request by moeddami**
See #45 

#### PR checklist
- [x] String input
- [x] File input
- [x] Process input line by line
- [x] Ignore input empty lines flag
- [x] Ignore unmatched lines flag
- [x] Support Windows PowerShell 5.1
- [x] Support PowerShell Core for Windows, Linux, and MacOS
- [x] JSON output with optional indent option
- [x] CSV output with optional delimiter option
- [x] Formatted table as default output
- [x] PowerShell help file
- [ ] Unit tests
- [ ] ~~Build/publish scripts~~

#### Current Syntax
String input:

```
grok -Input -Filter [-OutputFormat] [-IgnoreEmptyLines] [-IgnoreUnmatched] [-CsvDelimiter] [-IndentJson]
grok -i -f [-o] [-e] [-u] [-d] [-j]
```

File input:
```
grok -Path -Filter [-OutputFormat] [-IgnoreEmptyLines] [-IgnoreUnmatched] [-CsvDelimiter] [-IndentJson]
grok -p -f [-o] [-e] [-u] [-d] [-j]
```

Get Version:
```
grok -Version
grok -v
```

-OutputFormat accepts csv or json. Formatted table output if not provided.

#### Run
Build `Grok.Net.PowerShell` project.
Import Grok module:
```
cd ./bin/(Debug|Release)/netstandard2.0
import-module ./Grok-Test.psd1
help grok
```

or alternatively install already published test module:
```
Install-Module -Name Grok-Test
```

### CODE REVIEW

File: src/Grok.Net.Powershell/Grok.Net.PowerShell.csproj [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <AssemblyName>Grok.Net.PowerShell</AssemblyName>
+        <RootNamespace>GrokNet.PowerShell</RootNamespace>
+        <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
```
Comment (Marusyk) [inline src/Grok.Net.Powershell/Grok.Net.PowerShell.csproj:6]: Can we use `.net6` rather than `net461`?

**Review by Marusyk — APPROVED** [review]
This is amazing work. Thank you

**Comment by Marusyk** [comment]
Would you be interested to add иuild/publish scripts in a separate PR?

File: src/Grok.Net.Powershell/Grok.Net.PowerShell.csproj (hunk 1 above)
Comment (moeddami) [reply src/Grok.Net.Powershell/Grok.Net.PowerShell.csproj:6]: I thought `net461` is needed to support Windows PowerShell 5.1 but I realized it's not. `.netstandard2.0` alone will be just fine.

**Comment by moeddami** [comment]
Sure.

**Review by Marusyk — APPROVED** [review]
Thank you very much for your contribution

**Pull request merged**